### PR TITLE
split up theorems implying conjunctions of properties

### PR DESCRIPTION
**Pull request by StevenClontz**
Opening as a proof of concept...

### CODE REVIEW

**Comment by StevenClontz** [comment]
I marked a couple theorems as redundant, there are likely more we could easily identify as redundant with this proposal. Per #232 I used `deprecatedBy` but had to comment out that data so the tests would pass.

**Comment by StevenClontz** [comment]
With my experiment editing StackExchange network posts to update URLs, my feelings on preserving URLs have softened somewhat, particularly as we've broken URLs before (search syntax changed at some point) and it wasn't hard to fix those cases.

**Comment by StevenClontz** [comment]
New theorem numbers were chosen to ensure no conflicts (and to make writing things easier by only needing to change one digit to split `T000abc` to `T001abc` or `T002abc`. I'm not married to these however and can do a full search/replace to update these IDs to be at the end of our current list.

I do think that the issue of bringing together related theorems is a problem that cannot be fully solved by numbering things nicely, at least, not while also promising to avoid renumbering existing things.

**Comment by prabau** [comment]
I understand that we don't want to renumber existing theorems, but thought it would be good to group new ones together.  Leaving it up to you then.  I'll take a closer look at the contents.

File: theorems/T000100.md [hunk 1]
```diff
@@ -3,9 +3,7 @@ uid: T000100
 if:
   P000008: true
 then:
-  and:
-  - P000002: true
-  - P000014: true
+  P000002: true
 converse:
 - T000101
```
Comment (prabau) [inline theorems/T000100.md:8]: Remove the converse here.
And in T101 add T1100 to the converse.

File: theorems/T000171.md [hunk 2]
```diff
@@ -3,9 +3,7 @@ uid: T000171
 if:
   P000035: true
 then:
-  and:
-  - P000003: true
-  - P000030: true
+  P000030: true
```
Comment (prabau) [inline theorems/T000171.md:6]: This file in now a complete duplicate of T12 (and furthermore T12 points to a correct justification in S&S, whereas Fig 7 mentioned here does not seem to be it).  Delete file?

On the other hand, how do we now get [Fully T4 ==> T2] ?

File: theorems/T000176.md [hunk 3]
```diff
@@ -3,9 +3,7 @@ uid: T000176
 if:
   P000075: true
 then:
-  and:
-  - P000073: true
-  - P000016: true
+  P000073: true
 refs:
 - mr: MR1077251
```
Comment (prabau) [inline theorems/T000176.md:8]: Not a problem on this line, but the last line should have "{{mr:...}}" to show up in the browser.

File: theorems/T001176.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  P000075: true
+then:
+  P000016: true
+refs:
+- mr: MR1077251
+  name: General topology I (Arkhangelʹskiĭ, Pontryagin)
+---
+
+By definition.
+See example 21, section 2.6 of {{MR1077251}}.
```
Comment (prabau) [inline theorems/T001176.md:13]: Should be "{{mr:...}}".

**Review by prabau — COMMENTED** [review]
Also, seems T104 was skipped.

File: theorems/T001176.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  P000075: true
+then:
+  P000016: true
+refs:
+- mr: MR1077251
+  name: General topology I (Arkhangelʹskiĭ, Pontryagin)
+---
+
+By definition.
+See example 21, section 2.6 of {{MR:1077251}}.
```
Comment (prabau) [inline theorems/T001176.md:13]: need lowercase "mr"

File: theorems/T000176.md [hunk 6]
```diff
@@ -3,13 +3,11 @@ uid: T000176
...
   name: General topology I (Arkhangelʹskiĭ, Pontryagin)
 ---
 
 By definition.
-See example 21, section 2.6 of {{MR1077251}}.
+See example 21, section 2.6 of {{MR:1077251}}.
```
Comment (prabau) [inline theorems/T000176.md:13]: Does not show correctly in the browser. Seems it needs lowercase "mr".

Comment (prabau) [reply theorems/T000176.md]: fixed

File: theorems/T001176.md (hunk 5 above)
Comment (prabau) [reply theorems/T001176.md]: fixed

**Comment by prabau** [comment]
I think the only remaining thing is T04.

File: theorems/T000171.md (hunk 2 above)
Comment (StevenClontz) [reply theorems/T000171.md]: image

**Comment by StevenClontz** [comment]
I think it's done, but now we should consider compactifying the theorem IDs.

**Comment by prabau** [comment]
Great.  Let me take one last quick look and I'll approve.

**Review by prabau — APPROVED** [review]

**Comment by prabau** [comment]
Interesting, I clicked on "Squash and merge" (first time for me) and it shows the text of the combined commit, allowing to change it, like removing the irrelevant intermediate commit descritions, like "fix typos", update specific theorem, etc.  I then clicked on Cancel.  Leaving it up to you, but feel free to trim down the overall commit msg to make the git history easier to read.

**Comment by prabau** [comment]
Sorry, too late.

**Pull request merged**